text/0013-delete-enqueued-updates.md

```diff
@@ -0,0 +1,142 @@
+
+### Motivation
+
+The update queue is convenient because it allows us to serialize updates and assure the [ACID property](https://en.wikipedia.org/wiki/ACID) of MeiliSearch. It also causes some problems. As the queue message is an append-only queue, it is impossible to remove an update from this queue. The updates can be large and can take a long time to index. Sometimes, it would be convenient to abort a message in this queue or empty it. 
+
+### Additional Materials
+
+As explained on the [parent spec](https://github.com/meilisearch/specifications/pull/12), It's quite rare to have this kind of update queue. Elasticsearch or Algolia, who have this kind of update queue, don't give the abort or clear their 'tasks' queues. 
+
+### Explanation
+
+Two new routes and a new status (`aborted`) must be added to make this feature possible. 
+
+An aborted update will stay at its place in the queue. So the typical answer of the get all update status route will be: 
+
+```json
+[
+    {
+        "status": "processed",
+        "updateId": 1,
+        ...
+    },
+    {
+        "status": "failed",
+        "updateId": 2,
+        ...
+    },
+    {
+        "status": "processed",
+        "updateId": 3,
+        ...
+    },
+    {
+        "status": "enqueued",
+        "updateId": 4,
+        ...
+    },
+    {
+        "status": "aborted",
+        "updateId": 5,
+        ...
+    },
+    {
+        "status": "enqueued",
+        "updateId": 6,
+        ...
+    },
+    ...
+]
+```
+
+the update status document will look like:
+
+```json
+{
+  "status": "aborted",
+  "updateId": 12,
+  "type": {
+    "name": "DocumentsAddition",
+    "number": 4
+  },
+  "duration": 0,
+  "enqueuedAt": "2019-12-07T21:16:09.623944Z",
+  "processedAt": "2019-12-07T21:16:09.703509Z" // Date of deletion
+}
+```
+
+#### HTTP API
+
+**Abort an update:**
+
+- Method: DELETE
+- Route: `/indexes/:index_uid/updates/:update_id`
+    - index_uid: The index uid
+    - update_id: The update identifier
+- Status code: 204
+- Response: No Body
+
+** abort all update:**
+
+- Method: DELETE
+- Route: `/indexes/:index_uid/updates`
+    - index_uid: The index uid
+- Status code: 204
+- Response: No Response
+
+#### Potential errors
+
+- The given update id does not exist: 
+    - Status code: `404`
+    - Error code: `update_not_found`
+    - Error description: `The requested update can't be retrieved. Either it doesn't exist, or the database was left in an inconsistent state.`
+
+- The given update id is a processed id: 
+    - Status code: `400`
+    - Error code: `invalid_update_id`
+    - Error description: `The requested update id exist but could not be aborted. A processing/processed/failed update cannot be aborted.`
+
+- Error during the abort process:
+    - Status code: `400`
+    - Error code: `abort_update_failed`
```

Comment:
```suggestion
    - Error code: `update_abortion_failed`
```
`abort` is a verb, I try to be consistent with our other errors like `dump_creation_failed` that use a noun before the `_failed` suffix.